docs/commands/ComObjActive.htm

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
 <title>ComObjActive - Syntax &amp; Usage | AutoHotkey v2</title>
-<meta name="description" content="The ComObjActive function retrieves a running object that has been registered with OLE." />
+<meta name="description" content="The ComObjActive function retrieves a registered COM object." />
```

Comment:
Microsoft uses the exact phrase "a running object that has been registered with OLE", so I'm not sure I'd consider this an "issue", but I prefer the more succinct wording anyway.